### DISCUSSION TITLE
Not Showing daily/hourly/minute income

### DISCUSSION THREAD

**Discussion by AbleT4908** [Chia-Network/chia-blockchain#8952] [Support]
999
![KakaoTalk_20211026_161321284](https://user-images.githubusercontent.com/93184655/138832479-e2ec3cba-2d1e-4125-b7f1-6c4d7487f5b5.jpg)

I'm here wish to have a little help and get an advice. On october 26th, I had a very well work on my computers, but suddenly it shows no income shown on chia farming program as the photo I have sent above. 

The internet in my office has no problem and the latest income was shown till 10:01:58.

any help?

**Comment by poiu1101** [comment]
me too

**Comment by AbleT4908** [comment]
damn we are getting confused~. Dont know how to fix yet...0
​
-----Original Message-----
